package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "mdc-web-catalog",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-scripts": "1.1.1"
+  },
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
```

Comment:
was this all generated by create-react-app? I've never seen deploy and predeploy actually used.

Reply:
This is all stuff for Github pages, I'll remove it and add it in the next PR